src/parser.rs

```diff
@@ -13,6 +13,8 @@ use library::*;
 type Reader = EventReader<BufReader<File>>;
 type Attributes = Vec<OwnedAttribute>;
 
+const EMPTY_CTYPE: &'static str = "/*EMPTY*/";
```

Comment:
Seems for signals separate classes not needed.
Used 111 types can has "object" c_type, and can filled with it (like `GdkWindow*`) , others 5 is array types and can be processed later.
Full list https://gist.github.com/EPashkin/b13bf3e8e4a6087a8078.
Seems no primitive types without c:type in signals.